kolibri/plugins/learn/assets/src/views/ContentPage.vue

```diff
@@ -345,4 +345,8 @@
     font-weight: bold;
   }
 
+  .content-carousel {
+    margin-bottom: 24px;
+  }
```

Comment:
`content-carousel` is a class defined in another component, which should be considered a private implementation detail of the other component. Vue uses [scoped CSS](https://vue-loader.vuejs.org/guide/scoped-css.html) to generally prevent this behavior, but in this case you were able to bypass it because `content-carousel` happens to be defined on the root element which is a loophole in the scoped system.

Reply:
@indirectlylit On going through the css I found out that the KPagecontainer has already set padding of 24px.
![image](https://user-images.githubusercontent.com/33183263/75792663-4c9b2380-5d94-11ea-888e-dd091f2c00c6.png)

Reply:
thanks - looks like we can close #5531 then!